developer-tools.md

```diff
@@ -389,26 +389,25 @@ Here are instructions on profiling Spark applications using YourKit Java Profile
 <a href="https://www.yourkit.com/download/index.jsp">YourKit downloads page</a>. 
 This file is pretty big (~100 MB) and YourKit downloads site is somewhat slow, so you may 
 consider mirroring this file or including it on a custom AMI.
-- Untar this file somewhere (in `/root` in our case): `tar xvjf yjp-12.0.5-linux.tar.bz2`
-- Copy the expanded YourKit files to each node using copy-dir: `~/spark-ec2/copy-dir /root/yjp-12.0.5`
+- Unzip this file somewhere (in `/root` in our case): `unzip YourKit-JavaProfiler-2017.02-b66.zip`
+- Copy the expanded YourKit files to each node using copy-dir: `~/spark-ec2/copy-dir /root/YourKit-JavaProfiler-2017.02`
```

Comment:
Could we still use `spark-ec2/copy-dir`? (not sure to keep `spark-ec2` updated)

Reply:
Hi @shivaram, if I understood correctly, `amplab/spark-ec2` is managed by you. Would you mind if I ask whether we should keep this here? The change itself looks fine if I haven't missed something but I am less sure if we should keep this here or not.

Reply:
I think its probably good to add some wording on top saying `These instructions apply to when Spark is run using the spark-ec2[link to amplab/spark-ec2] scripts`

Reply:
Since this yourkit usage is not only for `spark-ec2`, I think the instruction might make others misunderstood. WDYT? @HyukjinKwon

Reply:
Actually, that's what I was thinking in what way too. But I would rather avoid to fix it here now as I guess it maybe needs to wait for some more opinions / discussion (e.g., other profiles tools, description for copying it to other nodes and checking duplication). I am fine with it if any committer strongly prefers (I am +0).

Otherwise, let's push what we are sure of first as the current changes look obviously something to be fixed. I don't want to make this PR complicated for now and the original intention looks describing yourkit with ec2 anyway.